ClimbingGearBackend/Controllers/UserGearController.cs

```diff
@@ -20,10 +20,9 @@ public UserGearController(ClimbingGearContext context)
 
     // GET: api/UserGear
     [HttpGet]
-    public async Task<ActionResult<IEnumerable<UserGearDTO>>> GetUserGear(long userId)
+    public async Task<ActionResult<IEnumerable<UserGearDTO>>> GetUserGear()
```

Comment:
What is the reason for this endpoint? Allowing anyone to see everyone's gear doesn't make sense.

Reply:
I just wanted it in a working state so I know it works. It's a simple matter to lock the whole controller and add a query that look for the current user and returns their gear.